tests/CommitCombinedStatus.py

```diff
@@ -33,8 +33,10 @@ class CommitCombinedStatus(Framework.TestCase):
     def setUp(self):
         super().setUp()
         self.combined_status = (
-            self.g.get_repo("edx/edx-platform", lazy=True)
+            self.g.withLazy(lazy=True)
+            .get_repo("edx/edx-platform")
             .get_commit("74e70119a23fa3ffb3db19d4590eccfebd72b659")
+            .complete()
```

Comment:
Is this actually needed? Wont the next method call cause the object to be completed anyway?

Reply:
This is just to keep the recorded test data. I could simply remove `.complete()` (getting the commit), which simplifies the test and proves the laziness works as expected: 8e02440aa9affc689cf23b74f461e9dd9a7ad543